scripts/esb_sniffer/README.rst

```diff
@@ -0,0 +1,90 @@
+.. _esb_sniffer_scripts_docs:
+
+ESB Sniffer
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Python scripts introduced in this document are used for the :ref:`esb_monitor` sample configured as ESB Sniffer.
+
+Set up Linux
+************
```

Comment:
Match the structure of other script files, for example https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/scripts/nrf_provision/fast_pair/README.html

- Title
- Shortdesc
- Overview (optionally, with additional details compared to Shortdesc)
- Requirements (path to the directory, prerequisites, such as Wireshark installation)
- Using the script
- Dependencies